### PR TITLE
Fixed returned values getLimit and getActivePage selectors.

### PR DESCRIPTION
**Pull request by stepek**
ESI-16485

### CODE REVIEW

File: src/TableView/TableSelectors.js [hunk 1]
```diff
@@ -107,7 +107,7 @@ export const getOffset = createSelector(
 
 export const getActivePage = createSelector(
   [offset, limit],
-  (offset, limit) => limit !== 0 ? Math.ceil(offset / limit) : 0
+  (offset, limit) => (limit !== 0 && limit !== undefined) ? Math.ceil(offset / limit) : 0
```
Comment (sealionkat) [inline src/TableView/TableSelectors.js]: It's better to check `limit !== undefined` first :)

**Comment by thayashi** [comment]
LGTM.
Regarding URL parameters, currently it uses offset, but I think it's better to use page=N like v1 style. Please think about this in API refactor @stepek

**Comment by thayashi** [comment]
screen shot 2018-01-03 at 8 16 03 am

**Comment by stepek** [comment]
@thayashi Sure

**Pull request merged**